src/main/scala/dsptools/counters/Counters.scala

```diff
@@ -66,21 +66,21 @@ class CountIO (countParams: CountParams) extends Bundle {
   // Count up/down control signal
   val upDown = if (countParams.countType == UpDown) Some(Input(Bool())) else None
   // Counters usually increment by 1
-  val inc = if (countParams.incMax != 1) Some(Input(UInt(countParams.incMax))) else None
```

Comment:
For automatically removing literals from IO Bundles: unclear whether this is a good idea. In ucb-bar/chisel3#168 rocket uses the same notation, but using the literal as a type (shorthand for not re-writing the bitwidth).